pkg/providers/nutanix/config/cp-template.yaml

```diff
@@ -0,0 +1,196 @@
+piVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: NutanixCluster
+metadata:
+  name: "{{.clusterName}}"
+  namespace: "{{.eksaSystemNamespace}}"
+spec:
+  prismCentral:
+    address: "{{.nutanixEndpoint}}"
+    port: {{.nutanixPort}}
+    insecure: true
+    credentialRef:
+      name: "{{.clusterName}}"
+      kind: Secret
+  controlPlaneEndpoint:
+    host: "{{.controlPlaneEndpointIp}}"
+    port: 6443
+---
+apiVersion: v1
+kind: Secret
```

Comment:
Instead of keeping the creds in this file, which gets written to disk, can we have it in a different file where we can just apply it from bytes?
Here's how we handle vSphere creds for reference https://github.com/aws/eks-anywhere/blob/main/pkg/providers/vsphere/config/secret.yaml#L1-L9